apps/web/components/ui/ImpersonatingBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import { useSession } from "next-auth/react";
+import { Trans } from "next-i18next";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Alert } from "@calcom/ui/Alert";
+
+type Props = {};
+
+function ImpersonatingBanner({}: Props) {
+  const { t } = useLocale();
+  const { data } = useSession();
+
+  if (!data?.user.impersonatedByUID) return null;
+
+  return (
+    <Alert
+      severity="warning"
+      title={<>{t("impersonating_user_warning", { user: data.user.username })}</>}
+      message={
+        <Trans i18nKey="impersonating_stop_instructions">
+          To stop impersonating
+          <a href="/auth/logout" className="underline">
+            Click Here
+          </a>
+          .
+        </Trans>
+      }
```

Comment:
Let's leave this in one line. Also made it briefer: 
<img width="455" alt="image" src="https://user-images.githubusercontent.com/3504472/165211792-4ca25112-48cb-4607-a09c-7108e3135360.png">